xml/net/server/protocol.xml

```diff
@@ -2252,7 +2252,9 @@
         <field name="npc_index" type="short"/>
         <field name="damage" type="three"/>
         <field name="hp_percentage" type="short"/>
-        <field name="kill_steal_protection" type="NpcKillStealProtectionState" optional="true"/>
+        <field name="kill_steal_protection" type="NpcKillStealProtectionState" optional="true">
+            <comment>This should always be sent to the attacker but may be left out for nearby players</comment>
```

Comment:
```suggestion
            <comment>The official server sends this field to the attacker, but not nearby players</comment>
```

Reply:
I don't want to make statements about what the client "expects" here, particularly when the client still **accepts** the packet and displays the attack action even if it produces an adverse effect.
I'd rather make statements about what we can observe of the protocol data, rather than potential bugs in the official client implementation.

We could simplify this even more though and say:
`This field should be sent to the attacker, but not nearby players.`

Reply:
@Cirras done